Survey/Configurations/Keychain/KeychainService.swift

```diff
@@ -40,7 +40,7 @@ final class KeychainService: KeychainServiceProtocol {
     }
 
     func isLoggedIn() throws -> Bool {
-        try keychain.get(KeychainKeys.token.rawValue) != nil
+        return try keychain.get(KeychainKeys.token.rawValue) != nil
```

Comment:
Please remove return here 😀 
```suggestion
       try keychain.get(KeychainKeys.token.rawValue) != nil
```